Source/IntegrationTests/TestFiles/LitTests/LitTest/vstte2012/BreadthFirstSearch.dfy.expect

```diff
@@ -1,2 +1,2 @@
 
-Dafny program verifier finished with 14 verified, 0 errors
+Dafny program verifier finished with 164 verified, 0 errors
```

Comment:
```suggestion
Dafny program verifier finished with 164 assertions verified, 0 errors
```
Is my understanding correct?

Reply:
I think it would be good to be explicit about the subject, but not in this PR, because it would be a massive change that would pollute the diff. Since the current absolute number has no meaning and it's only updated in a few places, I think it's OK not to add a subject in this PR.